### DISCUSSION TITLE
Encoders and motorfaders

### DISCUSSION THREAD

**Discussion by FoolProcessor** [shanteacontrols/OpenDeck#256] [Q&A] [answered]
Hey guys,
I just bought my first OpenDeck Platform L for a controller that fits my live music needs. As some parameters will be randomized from time to time on software side I was wondering if it is possbile to use motorfaders on platform L which "catch" the randomized software values. I just found a thread containing information about HDD motors.
Or is it possible to make encoders "catch" these values. So that I can alter them from the randomized point on.
As one might notice I am pretty new to this whole topic. So any hint is appreciated.
Thanks in advance
FoolProcessor

**Comment by paradajz** [comment, marked as answer]
Hi,

motorized faders aren't supported. For encoders, there is a "sync" option which you can use.
https://github.com/shanteacontrols/OpenDeck/wiki/Configurable-features#remote-cc-sync

**Reply by FoolProcessor** [reply]
Thank you.
This is what I was looking for.
